TeamCharles/initial-site
# 37-delete-edit-icons :: Add Edit and Delete Icons to Products owned by Current User

## Status

**READY**
## Documentation

[X] I have fully documented features in this PR. Please put an `X` in the box to confirm.
## Migrations

NO
## Description

Added a Delete and Edit icon for products that match a specified UserId. I am submitting a separate issue to update the view once we have ActiveUser stuff figured out.
## Related PRs
## Resolves Issue Number

resolves #37 
## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
## Steps to Test or Reproduce

``` sh
git pull --prune
git checkout 37-delete-edit-icons
dotnet run
```
1. Check out the refrigerator
## Impacted Areas in Application

List general components of the application that this PR will affect:
- Product list
